src/collection-view.js

```diff
@@ -538,6 +538,8 @@ const CollectionView = Backbone.View.extend({
 
     _.each(views, view => {
       renderView(view);
+      // corresponds that this view is not a pre-rendered one
```

Comment:
I don't think that the comment is necessarily true.  This flag indicates that the view is attached to something else in marionette.  where as `_isAttached` only indicates that the view is attached to the DOM.  All prerendered views shown in a region or collection view will also have `_attachedTo = true`